build.gradle

```diff
@@ -9,6 +9,8 @@
 plugins {
     // Apply the java plugin to add support for Java
     id 'java'
+    id("com.github.ben-manes.versions") version "0.46.0"
+    id 'se.patrikerdes.use-latest-versions' version '0.2.18'
```

Comment:
nit: lack of consistency. I think in .gradle file the convention is to not use parenthesis.

Reply:
updated